src/libraries/System.Console/tests/Color.cs

```diff
@@ -32,6 +34,15 @@ public static void RoundtrippingColor()
     }
 
     [Fact]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public static void RoundtrippingColor_Browser()
+    {
+        Assert.Throws<PlatformNotSupportedException>(() => Console.BackgroundColor = Console.BackgroundColor );
+        Assert.Throws<PlatformNotSupportedException>(() => Console.ForegroundColor = Console.ForegroundColor );
+    }
```

Comment:
Is this testing that the setter or the getter throws?  As written, we can't tell.  Assuming both throw, it'd be better to have four separate asserts:
```C#
Assert.Throws<PlatformNotSupportedException>(() => Console.BackgroundColor);
Assert.Throws<PlatformNotSupportedException>(() => Console.ForegroundColor);
Assert.Throws<PlatformNotSupportedException>(() => Console.BackgroundColor = ConsoleColor.Red);
Assert.Throws<PlatformNotSupportedException>(() => Console.ForegroundColor = ConsoleColor.Red);
```
at which point the test should also be renamed appropriately.

Reply:
Divided into separate asserts